app/src/main/java/org/mozilla/focus/fragment/AddToHomescreenDialogFragment.kt

```diff
@@ -62,6 +62,7 @@ class AddToHomescreenDialogFragment : DialogFragment() {
         warning.visibility = if (blockingEnabled) View.GONE else View.VISIBLE
 
         val editableTitle = dialogView.findViewById<EditText>(R.id.edit_title)
+        editableTitle.requestFocus()
```

Comment:
It was a small bug. When the dialog pops up the edittext isn't focused, so as a side effect you don't see the accent color as the underline. I can move it out into a separate PR, but it would just be this one line change.